src/matlab2tikz.m

```diff
@@ -2600,6 +2602,18 @@ function assertRegularAxes(handle)
             plotOptions = opts_add(plotOptions,'contour/labels','false');
         end
 
+        % Get line properties
+        [m2t, lineOptions] = getLineOptions(m2t, h);
```

Comment:
Yep, read the diff wrong. Looks good. 

We should edit some contours to cover the cases in #867